pkg/events/events.go

```diff
@@ -6263,6 +6264,24 @@ var Definitions = eventDefinitions{
 				{Type: "dev_t", Name: "dev"},
 			},
 		},
+		SecurityBpfProg: {
+			ID32Bit: sys32undefined,
+			Name:    "security_bpf_prog",
+			Probes: []probeDependency{
+				{Handle: probes.SecurityBpfProg, Required: true},
+				{Handle: probes.BpfCheck, Required: true},
+				{Handle: probes.CheckHelperCall, Required: false},
+				{Handle: probes.CheckMapFuncCompatibility, Required: false},
```

Comment:
Why are these two not required? Do these functions exist in all kernel versions?

Reply:
thanks @NDStrahilevitz ! you're right.

it is documented here: https://github.com/aquasecurity/tracee/blob/main/docs/docs/events/builtin/extra/bpf_attach.md#issues

just to elaborate:
not in all kernels these symbols exists. in some kernels these symbols could be optimized by the compiler, and it'll change the symbol name accordingly (e.g. check_helper_call.isra.0). therefore we use both of them - to cover all cases.

this behavior can result in error message from libbpf.
but we can skip this error using the logger, can't we? (i though we already did that: https://github.com/aquasecurity/tracee/blob/main/pkg/cmd/initialize/callbacks.go#L24)